src/commands/publish/upload_form/project_assets.rs

```diff
@@ -58,7 +60,7 @@ impl ProjectAssets {
         self.script_name.to_string()
     }
 
-    pub fn script_path(&self) -> String {
-        self.script_path.to_string()
+    pub fn script_path(&self) -> PathBuf {
```

Comment:
if this is not expected to change, should we return a reference to the attribute itself?

Reply:
yes!!! i think we are on the same page :)